testimony.yaml

```diff
@@ -25,8 +25,8 @@ CaseComponent:
     - Branding
     - BVT
     - Candlepin
-    - Capsule
-    - Capsule-Content
+    - ForemanProxy
+    - ForemanProxy-Content
```

Comment:
Rename of `Capsule - Content` to `Foreman Proxy - Content` was cancelled

There will be:
- `ForemanProxy`  (rename)
- Capsule-Content` (no rename)

Reply:
@lpramuk Yeah I saw that conversation but the ohsnap PR is merged as is means the `ForemanProxy-Content` is still appearing in ohsnap!

Reply:
OK. We need to be on par with `satellite/ohsnap!1066` MR.